biolearn/data_library.py

```diff
@@ -443,6 +621,7 @@ def __init__(self, data, cache=None):
             if getattr(self, field) is None:
                 raise ValueError(error_message)
         self.cache = cache if cache else NoCache()
+        self.data = data
```

Comment:
If were going to put  this on the DataSource then we should use a more informative name. Data is confusing since this class gets data for us. Recommend "definition" or "source_defenition"

Reply:
Done.